Source/Networking/GDataProgressMonitorInputStream.h

```diff
@@ -47,11 +47,11 @@
   unsigned long long dataSize_;     // size of data in the source
   unsigned long long numBytesRead_; // bytes read from the input stream so far
 
-  __weak id monitorDelegate_;
+  id monitorDelegate_;
```

Comment:
Can you throw a comment on here to still capture that it is weak?

  id monitorDelegate_;  // weak